carma-messenger-ui/website/widgets/eventManagement/widget.js

```diff
@@ -299,6 +308,7 @@ CarmaJS.WidgetFramework.eventManagement = (function () {
               newInputUpTrack.name = 'UpTrack';
               newInputUpTrack.className="form-control col-2";
               newInputUpTrack.id = 'UpTrack';
+              newInputUpTrack.type = "number";
```

Comment:
Input for advisory speed should also be number right? I only see it for uptrack/downtrack/min_gap